### PR TITLE
feat: make the image repo/username/name configurable

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**

### CODE REVIEW

File: helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
       {{- end }}
       containers:
         - name: move2kubeapi
-          image: "quay.io/konveyor/move2kube-ui:{{ .Values.deployment.api.imageTag | default .Chart.AppVersion }}"
+          image: "{{ .Values.deployment.api.imageName }}:{{ .Values.deployment.api.imageTag | default .Chart.AppVersion }}"
```
[the remark this answers is not in the document]
Comment (seshapad) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: @ashokponkumar should there be default for the image tag as well?

[the remark this answers is not in the document]
Comment (ashokponkumar) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: the chart appV?erison is the default there already.

[the remark this answers is not in the document]
Comment (seshapad) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: Missed that. Thanks!

[the remark this answers is not in the document]
Comment (HarikrishnanBalagopal) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: we have a default in the `values.yaml`

[the remark this answers is not in the document]
Comment (ashokponkumar) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: Why not have it here?

[the remark this answers is not in the document]
Comment (HarikrishnanBalagopal) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: ```Why not have it here?```
That's not the right place for it.

[the remark this answers is not in the document]
Comment (ashokponkumar) [reply helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml:33]: This is a default tha tomato users should not change. Those which most users should configure should go in values.yaml, and others can go in the respective yamla. It also allows users who are looking at the yamls to have complete information.

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**